classes/TaxonomyUtilities.php

```diff
@@ -230,6 +245,12 @@ public static function parseScientificName($inStr, $conn = null, $rankId = 0, $k
 			}
 			$sciname .= $retArr['unitname2'].' ';
 			$sciname .= trim($retArr['unitind3'].' '.$retArr['unitname3']);
+			if(!empty($retArr['cultivarepithet'])){
+				$sciname .= ' "' . $retArr['cultivarepithet'] . '"';
```

Comment:
I think that this cultivarEpithet should be surrounded by single quotes rather than double quotes? Also, how do we handle the case where the end user has already entered single quotes around their string in the cultivarEpithet input field?

In other words, I think the same processing that happens in classes/TaxonomyEditorManager.php should probably happen here, too?

Reply:
Fixed in 360285873.